### ISSUE TITLE
Zwei Dumme ein Gedanke ;-)

### ISSUE THREAD

**Issue by NMarkgraf** [NMarkgraf/FallstudienWebseite#6]
Tja Karsten,

ein kleiner Commit-War um die Hoheit, wer hier die Reihenfolge ändert :-p

Ich hoffe, es passt nun.

Norman

**Comment by luebby** [comment]
Passt - glaube ich

**Comment by NMarkgraf** [comment]
Nun ja, nach dem nächsten commit von mir ist der letzt toml-bug auch raus. Dann schon! - Danke Karsten für die ganzen tollen Fallstudien! -- Ich hoffe mein Gemeckere hällt dich nicht davon ab, noch mehr einzustellen! ;-)

**Comment by luebby** [comment]
Einzig: jetzt zwei lila nebeneiander, aber ich denke da ist @sebastiansauer dran (?)

**Comment by NMarkgraf** [comment]
Richtig! - Die aktuellen HIntergründe sind ja nur Platzhalter für die, die @sebastiansauer  da noch zaubern wird.

**Comment by sebastiansauer** [comment]
Bin dran, kommt, viel um die Ohren
